lib/ttnt/testtask.rb

```diff
@@ -65,7 +65,7 @@ def define_tasks
     def define_run_task
       desc @run_description
       task 'run' do
-        target_sha = ENV['TARGET_SHA'] || repo.head.target_id
+        target_sha = ENV['TARGET_SHA'] || nil
```

Comment:
Oh, you are right. Fixed: https://github.com/Genki-S/ttnt/commit/21291732c1fa33f5a901513d7b8ff1514b8d9686